addons/im_livechat/models/chatbot_script_question_answer.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models, fields
+from odoo.osv import expression
+
+
+class ChatbotScriptQuestionAnswer(models.Model):
+    _name = 'im_livechat.chatbot.script_question_answer'
+    _description = 'Chatbot Script Question Answer'
+
+    name = fields.Char(string='Name')
+    sequence = fields.Integer(string='Sequence')
+    step_id = fields.Many2one(
+        'im_livechat.chatbot.script_step', string='Script Step', required=True, ondelete='cascade')
+    chatbot_id = fields.Many2one(related='step_id.chatbot_id', store=True)
+
+    # TODO PKO: Test if conflict with name_get ? not sure why we defined name_get in the first place...
+    def _compute_display_name(self):
```

Comment:
You should not have to override this normally, name_get does the job and is the correct way of defining a custom display name for a record.

Reply:
Actually I overrode it for the `many2many_tags` to show only the name of the answer and not the step message followed by the answer name as it's done with the `name_get` method

Reply:
Well that's strange ... you would better rely on a context key used in the m2m and used in ``name_get`` then I think, if I understood the issue. Display name is basically a reflexion of name_get, so I would avoid mixing both methods overrides.